setup/nextcloud.sh

```diff
@@ -62,6 +62,9 @@ user_external_hash=280d24eb2a6cb56b4590af8847f925c28d8d853e
 # 5.3 You still can create, edit and delete users
 # 5.4 Go to Administration > Logs and ensure no new errors are shown
 
+# Set a local variable for the PHP version
+NC_PHP_VER=PHP_VER
```

Comment:
`NC_PHP_VER=$PHP_VER`

Reply:
Good catch. Fixed it.
At first I was puzzled why my test was succeeding. Because my testing only does a fresh installation, the variable is not used yet. I will encounter issues like this in the upgrade testing.